client/stylesheets/_modal.scss

```diff
@@ -902,6 +902,19 @@ div[class="sp-replacer sp-light full-spectrum sp-active"] {
 	height: 100px;
 }
 
+.modFeedbackInitial button[data-target="#feedbackBuyerUnsuccessfulModal"], .modFeedbackInitial button[data-target="#feedbackBuyerNeverModal"] {
+	padding: 11px 50px !important;
+	color: #fff !important;
+	background-color: #f8504b;
+	border: none;
+	border-radius: 4px !important;
+}
+
+.modFeedbackInitial button[data-target="#feedbackBuyerUnsuccessfulModal"]:hover, .modFeedbackInitial button[data-target="#feedbackBuyerNeverModal"]:hover {
+	background-color: #f73c37 !important;
```

Comment:
Color literals like `#f73c37` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>!important should not be used<br>Line should be indented with spaces, not tabs